slurm/README.gracc-fields

```diff
@@ -0,0 +1,78 @@
+EndTime                          March 26th 2019, 19:42:26.000
+GlobalJobId                      slurm:slurmdb-mgmt/3307/slurm/hipergator.34033351
+Grid                             OSG
+JobName                          bl_cms914c44d277b1
+LocalJobId                       34033351
+LocalUserId                      cmspilot
+Memory                           20480
+Memory_description               RSS
+Memory_metric                    total
+Memory_storageUnit               kB
+Njobs                            1
+ProbeName                        slurm:cms.rc.ufl.edu
+Processors                       8
+Processors_metric                total
+ProjectName                      avery
+Queue                            hpg1-compute
+RawXML                           (See below)
+RecordId                         cms.ufhpc:3444492.622
+ReportableVOName                 cms
+ResourceType                     Batch
+SiteName                         UFlorida-HPC
+StartTime                        March 26th 2019, 19:17:32.000
+Status                           0
+VOName                           /cms/uscms/Role=pilot/Capability=NULL
+WallDuration                     1494
+checksum                         ad6d91a9e96da9c6b444381f6d16f1bb
+lag_time                         89.16
+lag_time_hr                      0.025
+type                             JobUsageRecord
+
+
+The `RawXML` field contains all the info sent by the probe:
+
+<JobUsageRecord>
+<RecordIdentity urwg:createTime="2019-03-26T19:43:54Z" urwg:recordId="cms.ufhpc:3444492.622"/>
+<JobIdentity>
+        <GlobalJobId>slurm:slurmdb-mgmt/3307/slurm/hipergator.34033351</GlobalJobId>
+        <LocalJobId>34033351</LocalJobId>
+</JobIdentity>
+<UserIdentity>
+        <LocalUserId>cmspilot</LocalUserId>
+        <VOName>/cms/uscms/Role=pilot/Capability=NULL</VOName>
+        <ReportableVOName>cms</ReportableVOName>
+        <DN>/DC=ch/DC=cern/OU=computers/CN=cmspilot01/vocms080.cern.ch</DN>
+</UserIdentity>
+        <JobName>bl_cms914c44d277b1</JobName>
+        <Status>0</Status>
+        <Processors urwg:metric="total">8</Processors>
+        <WallDuration>PT24M54.0S</WallDuration>
+        <CpuDuration urwg:description="Was entered in seconds" urwg:usageType="user">PT34.92S</CpuDuration>
+        <CpuDuration urwg:description="Was entered in seconds" urwg:usageType="system">PT29.0S</CpuDuration>
+        <EndTime urwg:description="Was entered in seconds">2019-03-26T19:42:26Z</EndTime>
+        <StartTime urwg:description="Was entered in seconds">2019-03-26T19:17:32Z</StartTime>
+        <Queue>hpg1-compute</Queue>
+        <ProjectName>avery</ProjectName>
+        <Memory urwg:description="RSS" urwg:metric="total" urwg:storageUnit="kB">20480</Memory>
+        <ProbeName>slurm:cms.rc.ufl.edu</ProbeName>
+        <SiteName>UFlorida-HPC</SiteName>
+        <Grid>OSG</Grid>
+        <Njobs>1</Njobs>
+        <Resource urwg:description="ResourceType">Batch</Resource>
+</JobUsageRecord>
```

Comment:
The indentation seems off for this XML. And actually, I think this is the more useful information to break down in the README for the gratia probe. What I'm ultimately after is a description of the structure and fields of a gratia probe:

- Each record has info identifying the record, user, and job as well as info regarding job usage
- What are the human descriptions of each field?

Reply:
@brianhlin the GRACC [dev docs](https://opensciencegrid.org/gracc/dev-docs/raw-records/) go into this a bit, mainly though linking to this doc, which goes into far more detail than I'm sure you ever wanted: https://www.ogf.org/documents/GFD.98.pdf

Reply:
>The indentation seems off for this XML

I agree but apparently the XML actually looks like that.

>And actually, I think this is the more useful information to break down in the README for the gratia probe

Yes, since that's what's actually sent.  And actually if you just look at the xml, it's mostly self-explanatory.  But sure, we can document some slightly more human readable descriptions for the fields too.


Some links for future reference:

https://github.com/opensciencegrid/gratia-probe/blob/master/common/gratia/common/record.py
https://github.com/opensciencegrid/gratia-probe/blob/master/common/gratia/common/xml_utils.py
https://github.com/edquist/osg-twiki-backup/blob/cleanup/data/Accounting/ProbeDevelopement.txt

Reply:
Thanks @retzkek!  I did not know that super useful gracc/dev-docs/raw-records doc existed!

Reply:
@retzkek Thanks, that's really helpful!

> And actually if you just look at the xml, it's mostly self-explanatory.

Yeah, except when it's not :P. Like when we use the SiteName to actually refer to the topology resource name, or what does project name correspond to?

Reply:
>Like when we use the SiteName to actually refer to the topology resource name, or what does project name correspond to?

I think it's just the `SiteName` as defined in the `ProbeConfig`

Reply:
I'm more interested in the information that we intend to convey in these values rather than their provenance. For instance, if I give you a default `ProbeConfig`, what should you enter into the `SiteName` field?